tests/integration/test_ingest_markdown_docs_tool.py

```
@@ -0,0 +1,461 @@
+            assert call_args.kwargs["frontmatter_override"] == {
+                "product_line": "Check Point Endpoint",
+                "version": "R81.10",
+                "severity_level": "Medium",
+            }
+
+        finally:
+            os.unlink(tmp_file_path)
+
+    async def test_ingest_with_progress_tracking(self, mock_context, temp_markdown_files):
+        """Test progress tracking during batch processing."""
+        temp_dir, files = temp_markdown_files
```

Pylint (3.3.7)

[warning] Unused variable 'files'

(W0612)